app/scripts/models/reliers/relier.js

```diff
@@ -91,7 +94,49 @@ define([
      */
     allowCachedCredentials: function () {
       return this.get('allowCachedCredentials');
+    },
+
+    /**
+     * Creates a resume token from subset of model fields.
+     */
+    getResumeToken: function () {
+      var resumeObj = {};
+      var fieldCount = 0;
+
+      _.each(this.getRelierFieldsInResumeToken(), function (itemName) {
+        if (this.has(itemName)) {
+          resumeObj[itemName] = this.get(itemName);
+          fieldCount++;
+        }
+      }, this);
+
+      if (fieldCount === 0) {
+        return null;
```

Comment:
I added this `return null` here to prevent useless resume tokens from being included in the outgoing link; if we return an empty object then you get a `"resume=e30=` appended to the URL.  We could fix this elsewhere in the code though, which might make things a bit cleaner.